### ISSUE TITLE
 Insufficient color contrast ratios for borrow button

### ISSUE THREAD

**Issue by jdlrobson** [internetarchive/openlibrary#2610]
Spun out from #1005
Our borrow buttons (#3B98FC background, white text) are impossible for some users to read.

* Visit https://openlibrary.org/ in Google Chrome
* Open up the developer tools and click audits tab
* Tick the box for accessibility and click "run audits"

### Evidence / Screenshot (if possible)
The audit complains that 
"Background and foreground colors do not have a sufficient contrast ratio. Low-contrast text is difficult or impossible for many users to read. Learn more."
`a.cta-btn.cta-btn--available, a.cta-btn.cta-btn--available.borrow-link`

### Expectation
- [ ] button color should be changed to #0476B8 The tool https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=lighthouse will be helpful.
- [ ] The audit must pass.

**Comment by jdlrobson** [comment]
Current:
![Screenshot 2019-11-08 at 2 05 22 PM](https://user-images.githubusercontent.com/148752/68513813-d5fc7800-0230-11ea-80f9-2741423e1f76.png)


If we use #0476B8 
![Screenshot 2019-11-08 at 2 02 40 PM](https://user-images.githubusercontent.com/148752/68513697-8c139200-0230-11ea-9406-8dbd08b68254.png)

What do we think? @seabelis @mekarpeles @konpyuta @cdrini

**Comment by seabelis** [comment]
Agreed. I think the slight change in colour provides benefit without negatively impacting the visual presentation.

**Comment by jdlrobson** [comment]
Thanks for the feedback @seabelis 
I'll give people a week to feed back and if nothing let's move forward with fixing this.

**Comment by tfmorris** [comment]
I like the new color. Presumably we'd also change the color for the left/right nav buttons to match?

On a related note,iIt seems like there's also room to use a larger font for the button labels if we're trying to improve usability.

**Comment by konpyuta** [comment]
Color looks good to me!
I agree with @tfmorris , larger font size would benefit a lot.
Not sure if the left/right nav buttons really need to match though – they don't have the same contrast issues as the buttons with text and the slight difference between the nav buttons and the main action buttons might help the nav buttons not stand out too much. Worth a try to see how it looks with the same blue though.

**Comment by jdlrobson** [comment]
Anyone able to provide a patch? I will happily review :)

**Comment by lsarni** [comment]
@jdlrobson I'm working on a patch, I should be able to do the pull request today if the audit passes.
